CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Improvements:
 
 - Correct `PermissionRequestOption` typo with a class type alias.
   Which also raised the Dart SDK constraint to `2.13.0`.
+- The original predicate for filtering live-photo cannot be executed correctly and unexpectedly rejects photos with media subtype `PHAssetMediaSubtypeNone`. Changing it from `( ( mediaSubtype & 8 ) != 8 )` to `NOT ( ( mediaSubtype & 8 ) == 8` works correctly.
```

Comment:
We don't need a detailed explanation in the CHANGELOG. It should be placed around the updated code instead.

Reply:
Agree, how about this?
```suggestion
- The original predicate for filtering live-photo cannot be executed correctly and unexpectedly rejects photos with media subtype `PHAssetMediaSubtypeNone`. Changing it from `( ( mediaSubtype & 8 ) != 8 )` to `NOT ( ( mediaSubtype & 8 ) == 8` works correctly.
- Workaround for live-photo filtering bug in PhotoAsset framework on iOS.
```

Reply:
```suggestion
- Improve Live-Photos filtering.
```